guides/source/getting_started.md

```diff
@@ -2131,7 +2131,9 @@ following:
 <% end %>
 ```
 
-Then update `app/views/products/show.html.erb` to render this partial after the
+NOTE: `inventory_count?` is query method created by Active Record. Active Record adds query methods for all attributes. For numeric attributes these methods  return `true` for all non-zero number, and `false` otherwise.
```

Comment:
There is a minor typo (a missing "a"): NOTE: `inventory_count?` is **a** query method created by Active Record.

Reply:
Can we add a link to the docs @p8 added?